package.json

```diff
@@ -61,11 +61,14 @@
     "chai": "3.5.0",
     "conventional-changelog": "^1.1.0",
     "css-loader": "0.26.1",
+    "documentation": "^4.0.0-beta.18",
```

Comment:
They have a v3.0.5 which is from 2015, it looks like they have moved to v4.0.0 but didn't decided yet to introduce one stable version for that. Maybe I should open an issue ^^